wrappers/javascript/anoncreds-nodejs/package.json

```
@@ -47,7 +47,7 @@
   "binary": {
     "module_name": "anoncreds",
     "module_path": "native",
-    "remote_path": "v{version}",
+    "remote_path": "0.1.0-dev.1",
```

The other repos use a `v` prefix for the tag

```suggestion
    "remote_path": "v0.1.0-dev.1",
```